### ISSUE TITLE
bug: Duplicated Tutorials

### ISSUE THREAD

**Issue by MartinquaXD** [cowprotocol/docs#155]
### Problem
The tutorial https://beta.docs.cow.fi/cow-protocol/tutorials/integrations/scripts/how-to-submit-orders-via-the-api and https://beta.docs.cow.fi/cow-protocol/tutorials/integrations/scripts/submit-limit-orders-via-api/submitting-the-limit-order are completely identical and duplicate the content of https://beta.docs.cow.fi/cow-protocol/tutorials/integrations/scripts/sign-and-post-orders in other words.

So basically we have tutorials for the same thing 3 times.
